docs/spec/circuit-breaker/01_concepts.md

```diff
@@ -0,0 +1,17 @@
+# Concepts
+
+The intention of the circuit breaker is to have a contingency plan for a
+running network which maintains network liveness. This can be achieved through
+selectively "pausing" functionality of specific modules on a running network.
+The circuit breaker is intended to be enabled through either:
+
+ - governance, 
+ - the bonded validator group (for emergencies), 
```

Comment:
I suggest any set of accounts with varying power distribution which is selected by the state machine, instead of a partial validator set. For an emergency situation, there will not be enough time to make a full consensus from the entire validator set, but if the state machine selects some validators to give this permission, it will make the power distribution between the validators not proportional to the voting power and possibly create some vulnerability. 

I'm not sure which is the best way to handle these emergency situations but denoting that the emergency permission is not restricted on the validators will leave more possibilities.

Reply:
that makes a lot of sense, I'll integrate into the documentation